### DISCUSSION TITLE
No pointerover styling possible in TabControl

### DISCUSSION THREAD

**Discussion by NickLammerskitten** [AvaloniaUI/Avalonia#12988] [Q&A]
**Is your feature request related to a problem? Please describe.**
When I am styling my TabControl, the pseudoclass :pointerover is not shown in the frontend.

**Describe the solution you'd like**
I want to style the :pointerover different to the given FluentTheme styling.

**Describe alternatives you've considered**

**Additional context**
TabItem :selected ->
<img width="263" alt="Bildschirmfoto 2023-09-22 um 11 25 09" src="https://github.com/AvaloniaUI/Avalonia/assets/120336345/06a92caf-35f2-4ec3-a68c-46031bb60173">

TabItem :selected and :pointerover
<img width="217" alt="Bildschirmfoto 2023-09-22 um 11 30 17" src="https://github.com/AvaloniaUI/Avalonia/assets/120336345/b216f672-e6e2-4111-b606-26c45741de48">

Here you can see my code, which should set the foreground to red, when the :pointerover pseudoclass is activated:
`<Style Selector="TabItem:pointerover">
        <Setter Property="Foreground" Value="Red"/>
</Style>`

Neither the following has no result in the frontend:
`<Style Selector="TabItem:selected:pointerover">
        <Setter Property="Foreground" Value="Red"/>
</Style>`

**Comment by timunie** [comment]
please check original styles to understand which selector to be overridden. Most likely there is also a DynamicResource which can simply be overridden in App.resources. 

Original Style: https://github.com/AvaloniaUI/Avalonia/blob/d6ecb517b8dd2597aa72fb09cad65b299b8d9bb1/src/Avalonia.Themes.Fluent/Controls/TabItem.xaml#L66-L70

**Comment by timunie** [comment]
I'll convert it into a discussion for now. Let me know if it worked.
